roles/infrastructure/krb5_client/tasks/freeipa_autodns.yml

```diff
@@ -51,23 +51,4 @@
         dest: /etc/NetworkManager/conf.d/disable-resolve.conf-managing.conf
         backup: yes
 
-- name: Disable nm-cloud-setup if present
```

Comment:
If this is not included, EC2 instances will fall off the network when this service refreshes the config